### PR TITLE
Interlistdiversity

### PR DESCRIPTION
**Pull request by fberlinger**
Minor changes to enable InterListDiversity as an evaluation metric

### CODE REVIEW

File: mab2rec/pipeline.py [hunk 1]
```diff
@@ -620,7 +620,8 @@ def _validate_bench(recommenders, metrics, train_data, test_data, cv):
                                   RankingRecoMetrics.Precision,
                                   RankingRecoMetrics.Recall,
                                   RankingRecoMetrics.NDCG,
-                                  RankingRecoMetrics.MAP)),
+                                  RankingRecoMetrics.MAP,
+                                  DiversityRecoMetrics.InterListDiversity)),
```
Comment (bkleyn) [inline mab2rec/pipeline.py:624]: Any reason why we can't also add IntraListDiversity here? From what I recall this should also run fine.

**Comment by bkleyn** [comment]
I see some tests are failing due to the recent changes we made in MABWiser to vectorize predict Linear policies. 

I think it would be best to fix this in a separate PR (updating assert values, and updating reqs to use mabwiser>=2.7.0). If there aren't any objections I can take care of this today.

BTW, would be nice to set up CI scheduling as discussed before.

File: mab2rec/pipeline.py (hunk 1 above)
Comment (fberlinger) [reply mab2rec/pipeline.py:624]: I don't have a test case for it rn, that's why I didn't add it. Other than that I agree.

Comment (skadio) [reply mab2rec/pipeline.py:624]: while at it, it might be worth adding a test case and making this complete?

**Comment by bkleyn** [comment]
> I don't see any test files in the Changed Files list? Where are we testing for Inter/Intra and confirming the expected outputs?

Added a test to `test_pipeline.py` that tests the benchmark() functionality where Inter/Intra list metrics are now included.

**Comment by skadio** [comment]
Completely unrelated comment; it has been a while since I looked at this code. It makes more sense, to me now, to have collected the label column as "response_column" OR "binary_response_column" if we must be explicit about 0/1 nature, rather than "click_column". Not important, just a minor point to mention before I forget

**Comment by bkleyn** [comment]
> Completely unrelated comment; it has been a while since I looked at this code. It makes more sense, to me now, to have collected the label column as "response_column" OR "binary_response_column" if we must be explicit about 0/1 nature, rather than "click_column". Not important, just a minor point to mention before I forget

Agreed. That would improve consistency between Jurity and Mab2Rec that uses response_col/response.

**Pull request merged**